pyscriptjs/src/python/pyscript/__init__.py

```diff
@@ -628,181 +27,38 @@ class should be killed eventually.
     """
 
     loop = loop
+    run_until_complete = staticmethod(run_until_complete)
+    write = staticmethod(write)
 
-    @staticmethod
-    def run_until_complete(f):
-        run_until_complete(f)
 
-    @staticmethod
-    def write(element_id, value, append=False, exec_id=0):
-        write(element_id, value, append, exec_id)
+def __getattr__(attr):
+    if attr == "js":
+        global js
+        import js
+        from _pyscript_js import showWarning
 
-
-def _install_deprecated_globals_2022_12_1(ns):
-    """
-    Install into the given namespace all the globals which have been
-    deprecated since the 2022.12.1 release. Eventually they should be killed.
-    """
-
-    def deprecate(name, obj, instead):
-        message = f"Direct usage of <code>{name}</code> is deprecated. " + instead
-        ns[name] = DeprecatedGlobal(name, obj, message)
-
-    # function/classes defined in pyscript.py ===> pyscript.XXX
-    pyscript_names = [
-        "PyItemTemplate",
-        "PyListTemplate",
-        "PyWidgetTheme",
-        "add_classes",
-        "create",
-        "loop",
-    ]
-    for name in pyscript_names:
-        deprecate(
-            name, globals()[name], f"Please use <code>pyscript.{name}</code> instead."
+        showWarning(
+            "<code>pyscript.js</code> is deprecated, please use <code>import js</code> instead.",
+            "html",
```

Comment:
when you introduce a new deprecation, please write a comment which says what was the last version without the warning, so that we can easily keep track of it and remember to delete the next version